examples/persist-create/persist_create.md

```diff
@@ -0,0 +1,35 @@
+# Persist write - Create record/records
+
+The Ballerina persistence feature provide support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once model is defined, the client API is generated with resources based on the model. The generated
+API can be used to query and manipulate the persistent data in the application.
+The generated client API provides `post` resource function to persist record/records to the data store.
+
+> **Note:** This example uses the Ballerina tables as the data store. You can MySQL and Google Sheets as the data store as well. For more information, see [Supported Data Stores](/learn/supported-data-stores/).
+
+#### Initialize the project
+Execute the command below to initialize persistence in the project.
```

Comment:
Should we use `bal persist` instead of persistence?

Reply:
yes